pytorch-job/pytorch-job-crds/overlays/application/application.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: pytorch-job-crds
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: pytorch-job-crds
+      app.kubernetes.io/instance: pytorch-job-crds-v0.7.0
+      app.kubernetes.io/version: v0.7.0
+      app.kubernetes.io/component: pytorch
+      app.kubernetes.io/part-of: kubeflow
+      app.kubernetes.io/managed-by: kfctl
+  componentKinds:
+  - group: core
+    kind: Service
+  - group: apps
+    kind: Deployment
+  - group: core
+    kind: ConfigMap
```

Comment:
Manifests don't have a ConfigMap?

Reply:
@johnugeorge you're right. There are cases where we do a configMapGenerator, but not here.
@richardsliu could you remove lines 19,20?
if you use the gen-tests-targets.sh from #466 (eg copy to your ~)

~/gen-test-targets pytorch-job/pytorch-job-crds/overlays/application